models/door_biestheuvel/model.config

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" ?>
+<model>
+  <name>door_biestheuvel</name>
```

Comment:
I think we should remove Biesheuvel from the name. As the door will stay in the lab